trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py

```diff
@@ -197,85 +208,138 @@ def empty(config,
+                    proprotor.mass_properties.mass = propeller_mass + prop_hub_weight + prop_servo_weight
+                
+        lift_rotor_servo_weight = 0.0  
+        if nLiftRotors > 0:
+            if propulsor.identical_lift_rotors:
+                # Get reference properties for sizing from first lift_rotor (assumes identical)
+                liftrotor = rots[list(rots.keys())[0]]
+                liftmotor = rot_motors[list(rot_motors.keys())[0]]
+                rTip_ref     = liftrotor.tip_radius
+                bladeSol_ref = liftrotor.blade_solidity 
+                
+                
+                if liftrotor.variable_pitch:
+                    lift_rotor_servo_weight = 0.65 * Units.kg
+                
+                # Compute and add lift_rotor weights
+                lift_rotor_mass                = prop(liftrotor, maxLift / max(nLiftRotors - 1, 1))  * Units.kg
+                output.lift_rotors            += nLiftRotors * lift_rotor_mass
+                output.lift_rotor_motors      += nLiftRotors * liftmotor.mass_properties.mass
+                liftrotor.mass_properties.mass = lift_rotor_mass + lift_rotor_hub_weight + lift_rotor_servo_weight
+                
+            else:
+                for idx, lift_rotor in enumerate(propulsor.lift_rotors):
+                    liftrotor    = lift_rotor
+                    liftmotor    = rot_motors[list(rot_motors.keys())[idx]]
+                    rTip_ref     = liftrotor.tip_radius
+                    bladeSol_ref = liftrotor.blade_solidity  
+                    
+                    
+                    if liftrotor.variable_pitch:
+                        lift_rotor_servo_weight = 0.65 * Units.kg                  
+                    
+                    # Compute and add lift_rotor weights
+                    lift_rotor_mass                = prop(liftrotor, maxLift / max(nLiftRotors - 1, 1))  * Units.kg
+                    output.lift_rotors            += lift_rotor_mass
+                    output.lift_rotor_motors      += liftmotor.mass_properties.mass
+                    liftrotor.mass_properties.mass = lift_rotor_mass + lift_rotor_hub_weight + lift_rotor_servo_weight                    
+       
+        # Add associated weights 
+        output.servos += (nLiftRotors * lift_rotor_servo_weight + nThrustProps * prop_servo_weight)
+        output.hubs   += (nLiftRotors * lift_rotor_hub_weight + nThrustProps * prop_hub_weight)
+        output.BRS    += (prop_BRS_weight + lift_rotor_BRS_weight) 
+        
+
         maxLiftPower   = 1.15*maxLift*(k*np.sqrt(maxLift/(2*rho_ref*np.pi*rTip_ref**2)) +
-                         bladeSol_ref*AvgBladeCD/8*maxVTip**3/(maxLift/(rho_ref*np.pi*rTip_ref**2)))
+                         bladeSol_ref*AvgBladeCD/8*maxVTip**3/(maxLift/(rho_ref*np.pi*rTip_ref**2)))  # not sure how to adjust for multiple rotors/props, which value of rTip_ref etc. to use?
```

Comment:
this isn't used if there are multiple rotors anyways, you can just put it inside the if statement below for the tail rotor

Reply:
done